sys-auth/AusweisApp2/AusweisApp2-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake git-r3
+
+DESCRIPTION="Official authentication app for German ID cards and residence permits"
+HOMEPAGE="https://www.ausweisapp.bund.de/"
+EGIT_REPO_URI="https://github.com/Governikus/AusweisApp2.git"
+
+LICENSE="EUPL-1.2"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="
```

Comment:
You are missing dependencies. It's also linked against udev. Since it's not part of base profile, it needs to be added. See ```virtual/udev/udev```
```  NEEDED               libudev.so.1```

Reply:
Done